gitserver/testcases/ClassloadingTakeOne/Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+// Property accesses like `map.A` cause Groovy to try to load `map$A` and `map.A` as classes.
```

Comment:
Personally, I wouldn't add either of these examples. They were useful, but because all of the class loading happens when the script is compiled vs at runtime, it's kind of hard to create a good test case to reproduce problems in a Hydra-like environment using the property expression quirk, and it doesn't really make sense to have the `System.gc()` call for this kind of test in general, since that is only relevant for the very specific bug addressed by https://github.com/jenkinsci/script-security-plugin/pull/265.

Reply:
I've got the examples saved elsewhere, come to think of it. If these are unlikely to be useful again, I can delete them from here.